lib/engine/game/g_1862/step/buy_train.rb

```diff
@@ -24,6 +24,7 @@ def setup
 
           def actions(entity)
             return [] if entity != current_entity
+            return [] if @game.skip_round[entity]
```

Comment:
Is there a way to hook into round.next_entity! to skip the corporation there, rather than having to add logic to multiple steps?

Reply:
I don't think this works since a company can go bankrupt or merge (the two ways skip_round is used) in the middle of an OR turn.

Reply:
I think you can still call next_entity! when they go bankrupt or merge. No strong preference on how you do it, but it did seem tedious to put skip in each step.

Reply:
I did it that way in 1860 and it seemed to cause more than it's share of bugs. I'm sure I just did it wrong. I have to special case for receivership and LNWR formation on a step by step basis anyway (those changes aren't in here yet) so it's really not that big a deal for me.